frame/staking/src/migrations.rs

```diff
@@ -19,6 +19,65 @@
 use super::*;
 use frame_election_provider_support::SortedListProvider;
 use frame_support::traits::OnRuntimeUpgrade;
+use frame_support::dispatch::GetStorageVersion;
+
+pub mod v13 {
+	use super::*;
+	use frame_support::{pallet_prelude::ValueQuery, storage_alias};
+
+	#[storage_alias]
+	type StorageVersion<T: Config> = StorageValue<Pallet<T>, Releases, ValueQuery>;
+
+	#[derive(Encode, Decode, Clone, Copy, PartialEq, Eq, RuntimeDebug, TypeInfo, MaxEncodedLen)]
+	enum Releases {
```

Comment:
We can still keep the `enum Releases` in this file, not public, for decoding purposes.